TextConverters/DOCX/GNUmakefile

```diff
@@ -0,0 +1,47 @@
+#   GNUmakefile
+#
+#   Copyright (C) 2001 Free Software Foundation, Inc.
```

Comment:
As this is a new file, you should update the date an add yourself as the author.